manifests/v1beta1/installs/katib-cert-manager/kustomization.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+  - ../katib-standalone
+  # Cert-manager certificate for webhooks
+  - certificate.yaml
+images:
+  - name: docker.io/kubeflowkatib/katib-controller
+    newName: docker.io/kubeflowkatib/katib-controller
+    newTag: latest
+  - name: docker.io/kubeflowkatib/katib-db-manager
+    newName: docker.io/kubeflowkatib/katib-db-manager
+    newTag: latest
+  - name: docker.io/kubeflowkatib/katib-ui
+    newName: docker.io/kubeflowkatib/katib-ui
+    newTag: latest
+  - name: docker.io/kubeflowkatib/cert-generator
```

Comment:
I think we can remove this image since we are not using cert-generator image in the cert-manager install.

Reply:
Good catch!